ydb/library/yql/core/peephole_opt/yql_opt_peephole_physical.cpp

```diff
@@ -5468,7 +5468,7 @@ bool CollectBlockRewrites(const TMultiExprType* multiInputType, bool keepInputCo
 
         TExprNode::TListType funcArgs;
         std::string_view arrowFunctionName;
-        if (node->IsList() || node->IsCallable({"And", "Or", "Xor", "Not", "Coalesce", "If", "Just", "Nth", "ToPg", "FromPg", "PgResolvedCall", "PgResolvedOp"}))
+        if (node->IsList() || node->IsCallable({"And", "Or", "Xor", "Not", "Coalesce", "If", "Just", "Member", "Nth", "ToPg", "FromPg", "PgResolvedCall", "PgResolvedOp"}))
```

Comment:
AsStruct?

Reply:
The revision was irrelevant. [Force-pushed](https://github.com/ydb-platform/ydb/commit/568b6c8198a0e23471ffcd776180a50db98ce56a).